INSTALL.rst

```diff
@@ -58,8 +58,8 @@ To update the installation::
 
 Platform-specific notes follow below.
 
-a. Windows
-``````````
+**a. Windows**
```

Comment:
this line (and the next change) was raising a sphinx warning "inconsistent title level", causing the doc build to fail. I tried every type of title symbols which I knew of but to no avail (fortunately I could reproduce locally) so here is a quick and dirty fix...